### PR TITLE
Pushing 2nd diagnostic for 10/19/2016

### PR DESCRIPTION
**Pull request by matthewgsommers**
Respond to diagnostic

### CODE REVIEW

**Comment by matthewgsommers** [comment]
Note: 

I forgot to include a rating of the difficulty of the assignment. 
Comfort: 3
Clarity: 5

I struggled to formulate a response to question 5 in the given time, dedicated too much time to the other questions. 

I used my notes from last class and today's class.

File: diagnostic.md [hunk 1]
```diff
@@ -33,6 +41,7 @@ y = 10;
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
+Both 'x' and 'y' are now set to the value of '10'.
```
Comment (neugierige) [inline diagnostic.md:44]: actually, the value of x would be 3. 

```
 let x = 4; // x = 4
 let y = 3; // x = 3
 x = y; // x = 3
 y = 10; // y = 10
```

Comment (neugierige) [inline diagnostic.md:44]: also, "Do the variables change?  Do their values?"

File: diagnostic.md [hunk 2]
```diff
@@ -45,18 +54,22 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+The 1st 'weather' expression sets 'weather' equal to the value associated with "sunny"
+The 2nd 'weather' experssion sets the absolute value of 'weather' to 'sunny' as a string
```
Comment (neugierige) [inline diagnostic.md:58]: No. `weather === "sunny";` checks whether `weather` is exactly equal to `"sunny"`. This expression would evaluate to `true`.

File: diagnostic.md [hunk 3]
```diff
@@ -45,18 +54,22 @@ weather === "sunny";
...
-let answer = 'not empty';
+let answer = '1';
 
 while (answer !== '') {
   answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
+  if (answer === '1') {
    console.log("You guessed it!");
    break;
   }
```
Comment (neugierige) [inline diagnostic.md:75]: You're halfway there! The question also asked you to console.log a message to the user prompting them to enter a number between 1 and 10. Take a look at this solution, and let us know if you have questions.

```
const ask = require('../lib/ask.js');

let answer = '';

while (answer !== '5') {
  if (answer < 1 || answer > 10) {
   console.log('Please try a number between 1 and 10');
 } else if (answer !== '') {
   console.log('That\'s not it!');
 }
  answer = ask("Guess my secret? ");
}
console.log("You guessed it!");
```

Comment (neugierige) [reply diagnostic.md:75]: also, you were supposed to put this in bin/diagnostic.js so that you can run the code

**Comment by neugierige** [comment]
2/5. Hi Matt, please read over my comments and explanations, and be sure to answer ALL the questions posed.

**Pull request closed without merging**